services/comparison/tests/unit/test_reports_uploaded_count_diff.py

```diff
@@ -0,0 +1,173 @@
+from unittest.mock import MagicMock
+
+import pytest
+from shared.reports.resources import Report
+from shared.utils.sessions import Session, SessionType
+
+from services.comparison import ComparisonProxy
+from services.comparison.types import Comparison, FullCommit, ReportUploadedCount
+
+
+@pytest.mark.parametrize(
```

Comment:
When adding parametrize tests, I like to [add ids](https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids) for each test case to more clearly communicate what the test is checking.

Reply:
this is so cool... I didn't know about that. Thanks for sharing, will add and ID on those